core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -106,12 +106,8 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
       cmdToLaunch = cmdToLaunch ++ Seq[String]("-driverif", conf.h2oDriverIf.get)
     }
 
-    if (hc.getConf.h2oNodeWebEnabled) {
-      if (hc.getConf.contextPath.isDefined) {
-        cmdToLaunch = cmdToLaunch ++ Seq("-context_path", hc.getConf.contextPath.get)
-      }
-    } else {
-      cmdToLaunch = cmdToLaunch ++ Seq[String]("-J", "-disable_web")
```

Comment:
To be able to compare H2O-3 versions via REST call, I need to enable web on executors. @jakubhava if you agree with this approach, I will create a seperate PR deprecating related properties for the next fix release.

Reply:
Yes, I think this is fine. We just need to check our security docs.